### DISCUSSION TITLE
Is there a general place where we can look for plugins?

### DISCUSSION THREAD

**Discussion by cmd-save** [GrapesJS/grapesjs#3649] [General]
It would be great if we have a repository for all plugins for grapejs. I am specifically looking for one that is a data grid or data table. Does anyone has the idea for such a plugin?

**Comment by ronaldohoch** [comment]
Several days ago i've looking into all github, searching from grapesjs, it has 37 pages, but i haven't see any data-table component, but, we have this: https://github.com/Anubhavjain786/grapesjs-table looks promising but i haven't tested it before.

also, I'll be glad in help to improve the plugin list or create a new one.

**Reply by artf** [reply]
I'd really appreciate help with the official plugin list. It would be great to have a place where people can filter/search plugins.
If anyone is interested please [DM on twitter](https://twitter.com/artur_arseniev)

**Reply by ronaldohoch** [reply]
can't DM you yet 😅
![image](https://user-images.githubusercontent.com/2287371/142429667-36dec94f-694e-4f6d-9d12-959638a6e78b.png)

**Reply by artf** [reply]
Ops 😅 Can you check now?
